--- cadaver-0.23.3/configure.ac.neon0.30~	2014-05-16 00:37:15.099263344 +0200
+++ cadaver-0.23.3/configure.ac	2014-05-16 00:37:43.183536877 +0200
@@ -33,7 +33,7 @@ AC_ARG_ENABLE(netrc,
 AC_HELP_STRING([--disable-netrc], [enable .netrc support]),,
 enable_netrc=yes)
 
-NE_REQUIRE_VERSIONS([0], [27 28 29])
+NE_REQUIRE_VERSIONS([0], [27 28 29 30])
 
 dnl Don't enable zlib or ACL support in neon
 NEON_WITHOUT_ZLIB
